### PR TITLE
improvements for performance

### PR DESCRIPTION
**Pull request by creed-victor**
- use rpc nodes instead of ws where possible
- moved proposal details from modal to dedicated page (prevents downloading data from other proposals)
- fixes progress bar image (it respects % now)

task: https://taiga.sovryn.app/project/sovryn-light/task/673

also added some info about current state and requirements of voting:

Screenshot 2021-12-10 at 14 52 01

task: https://taiga.sovryn.app/project/sovryn-light/task/652

### CODE REVIEW

File: src/app/containers/BlockChainProvider/saga.ts [hunk 1]
```diff
@@ -53,33 +53,22 @@ function* setupSaga({ payload }: PayloadAction<ChainId>) {
...
     });
 
-    return () => {
-      blockTracker.stop().catch(_ => {
-        // BlockTracker assumes there is an outstanding event subscription.
-        // However for our tests we start and stop a BlockTracker in succession
-        // that triggers an error.
-      });
-    };
+    return () => {};
```
Comment (tiltom) [inline src/app/containers/BlockChainProvider/saga.ts:68]: Create `noop` constant (as we have in our frontend repo) and use it here.

File: src/app/containers/ProposalRow/index.tsx [hunk 2]
```diff
@@ -16,6 +10,7 @@ import { ProposalRowStateBadge } from '../../components/ProposalRowStateBadge';
 import { dateByBlocks } from '../../../utils/helpers';
 import { MergedProposal } from '../../hooks/useProposalList';
 import { bignumber } from 'mathjs';
+import { LoadableValue } from 'app/components/LoadableValue';
 
 interface Props {
```
Comment (tiltom) [inline src/app/containers/ProposalRow/index.tsx:15]:
```suggestion
interface IProposalRowProps {
```

File: src/app/containers/ProposalRow/index.tsx [hunk 3]
```diff
@@ -92,7 +81,12 @@ export function ProposalRow({ proposal }: Props) {
           <>
             <td className="font-montserrat max-w-sm truncate">
               <Linkify newTab={true}>
-                {String(proposal.id).padStart(3, '0')} • {created?.description}
+                {String(proposal.id).padStart(3, '0')} •{' '}
+                {loadingCreated ? (
+                  <em>Loading, please wait..</em>
```
Comment (tiltom) [inline src/app/containers/ProposalRow/index.tsx:86]:
```suggestion
                  <em>Loading, please wait...</em>
```

File: src/app/containers/ProposalRow/index.tsx [hunk 4]
```diff
@@ -134,7 +128,12 @@ export function ProposalRow({ proposal }: Props) {
           <>
             <td className="font-montserrat max-w-sm truncate">
               <Linkify newTab={true}>
```
Comment (tiltom) [inline src/app/containers/ProposalRow/index.tsx:130]: We have a duplicate on line `84`, pls extract it to a component (you can leave the new component in this file).

**Review by tiltom — CHANGES_REQUESTED** [review]

**Comment by Alitasovryn** [comment]
Screen Shot 2021-12-10 at 9 18 53 AM

**Comment by Alitasovryn** [comment]
Works fine

**Review by Alitasovryn — APPROVED** [review]

**Review by soulBit — APPROVED** [review]
Changes look good, proposals load almost 2x faster now 😄 

I made two small updates - renamed Quarums -> QuorumDetails, and styling for new details to match the rest of the page content

**Review by tiltom — APPROVED** [review]

**Review by JoeStorm99 — APPROVED** [review]

**Pull request merged**